templates/shopify-bigquery/assets/shopify.customers.asset.yml

```diff
@@ -1,8 +1,87 @@
 name: shopify_raw.customers
 type: ingestr
 
+description: |
+  This asset manages the ingestion of Shopify customer data into BigQuery. It captures comprehensive customer information 
+  including personal details, contact information, order history, marketing preferences, and address data. The asset 
+  includes data quality checks to ensure critical fields like customer ID and email are properly populated.
+
```

Comment:
I'd suggest not wrapping the lines here, instead let them do it in their editor of choice.

Reply:
meaning ?